packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -69,6 +69,7 @@ export enum SnapshotFormatSupportType {
  */
 export async function fetchSnapshot(
 	snapshotUrl: string,
+	// eslint-disable-next-line @rushstack/no-new-null
```

Comment:
I assume this warning is disabled for whole package. Can we enable this warning in ts config?

Reply:
this warning is only disabled for this line